UnitTests/GitCommands.Tests/Helpers/PathUtilTest.cs

```diff
@@ -347,6 +348,13 @@ public void DeleteWithExtremePrejudice_should_return_true_if_delete_successful()
         [TestCase("https://github.com/gitextensions/gitextensions", true)]
         [TestCase("https://github.com/gitextensions/gitextensions.git", true)]
         [TestCase("github.com/gitextensions/gitextensions.git", true)]
+        [TestCase("git clone https://github.com/gitextensions/gitextensions", true)]
+        [TestCase("HTTPS://MYPRIVATEGITHUB.COM:8080/LOUDREPO.GIT", true)]
+        [TestCase("git://myurl/myrepo.git", true)]
+        [TestCase("git clone https://github.com/gitextensions/gitextensions && cd gitextensions", true)]
+        [TestCase("github.com/gitextensions", false)]
+        [TestCase("github.com/gitextensions/gitextensions/pull/9018", false)]
+        [TestCase("git clone ssh://username@gerrit-server:/PROJECT", true)]
```

Comment:
Neither of these are correct inputs for `CanBeGitURL` method - because the answer to the question "_Can either of those be a git URL?_" is "_no_".
```cs
        [TestCase("git clone https://github.com/gitextensions/gitextensions", true)]
        [TestCase("git clone https://github.com/gitextensions/gitextensions && cd gitextensions", true)]
        [TestCase("github.com/gitextensions", false)]
        [TestCase("github.com/gitextensions/gitextensions/pull/9018", false)]
        [TestCase("git clone ssh://username@gerrit-server:/PROJECT", true)]
```

We can consider renaming the method to `ContainsGitUrl`, because then the answer becomes "yes". But then the signature must be changed as well to return the valid git URL... So we'll have to make it something like:
```cs
        public static bool TryExtractGitUrl(string url, out string gitUrl)
```

Reply:
I believe the comments are addressed